docs/spark-procedures.md

```diff
@@ -721,14 +721,15 @@ Creates a view that contains the changes from a given table.
-| `changelog_view`        |   | string | Name of the view to create                                                                                                                                                                                            |
-| `options`     |   | map<string, string> | A map of Spark read options to use                                                                                                                                                                                    |
-|`compute_updates`| | boolean | Whether to compute pre/post update images (see below for more information). Defaults to false.                                                                                                                        | 
-|`identifier_columns`| | array<string> | The list of identifier columns to compute updates. If the argument `compute_updates` is set to true and `identifier_columns` are not provided, the table’s current identifier fields will be used to compute updates. |
-|`remove_carryovers`| | boolean | Whether to remove carry-over rows (see below for more information). Defaults to true.                                                                                                                                 |
+| Argument Name        | Required? | Type                | Description                                                                                                                                                                                                          |
+|----------------------|----------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `table`              | ✔️ | string              | Name of the source table for the changelog                                                                                                                                                                           |
+| `changelog_view`     |   | string              | Name of the view to create                                                                                                                                                                                           |
+| `options`            |   | map<string, string> | A map of Spark read options to use                                                                                                                                                                                   |
+| `net_changes`        | | boolean             | Whether to output net changes (see below for more information). Defaults to false. |
```

Comment:
Nit: if we are trying to fix alignment here anyway for all the rows in this PR, should we make make 'required' and 'type' columns have same alignment throughout?

Reply:
fixed